tests/trainer/test_trainer.py

```diff
@@ -37,18 +37,10 @@ def test_init(self, config):
         trainer = Trainer(**config)
         assert isinstance(trainer, Trainer)
 
-    def test_model_ddp_wrapped(self, config, init_process_group):
-        # dist initialized first with init_process_group fixture
+    def test_model_ddp_wrapped(self, config):
```

Comment:
I dont see the corresponding change to the `Trainer` code being pushed?

Reply:
https://github.com/mosaicml/composer/blob/5cc81464f5ba7277a26180f436aefcdce02d9be9/composer/trainer/trainer.py#L525